package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jamsocket",
-  "version": "0.4.5",
+  "version": "0.4.6",
```

Comment:
We can drop this back to `0.4.5` and then use [these directions](https://www.notion.so/drifting/Publish-CLI-to-NPM-Steps-ff8af8b19e414cee9d650ec328535263) for updating the version and publishing it to NPM.